ProductDock/rbc3-library-fe
# RBN-85: Add new book form

### What did you implement?

Implemented a feature where clicking on Add Book button opens new page with Add book form, back button that leads user back to previous page, implemented form UI with responsive design, without 'add another book' section.

### How can we verify the changes?

Changes can be verified by running the app and checking '/add-books' page, as well as '/admin' page where clicking on 'Add a new book' button should lead to page with implemented form.
### More info

- Jira: [RBN-85](https://productdock.atlassian.net/browse/RBN-85)
- Figma: [Design](https://www.figma.com/design/x4SgW80BE24s9gAFHM3gXS/RBC---Novi-Sad%2C-2024?node-id=3875-28226&t=lUkNoGjAioHpmC11-4)
